### DISCUSSION TITLE
How to get list of elements present in list view?

### DISCUSSION THREAD

**Discussion by Harsha-Gelato** [appium/appium#17333] [General]
Hi All,

I am using appium flutter driver. I have a small question:
Suppose, there is a list view on the app screen and the elements present in the list all has keys/ids like:

- **language_1**
- **language_2**
- **language_3** and so on.

![Screenshot_20220729_092006](https://user-images.githubusercontent.com/103107117/183434166-709faeb3-2006-4a66-9aef-e4e6d56e075a.png)

Is it possible to get the lists of all such ids/keys and then perform click operation on each element?

**Comment by KazuCocoa** [comment]
Perhaps flutter_driver https://api.flutter.dev/flutter/flutter_driver/flutter_driver-library.html package does not provide such a function.
Then, FLUTTER context cannot get such list unlike other drivers, unfortunately.

**Comment by saravanakumarQA** [comment]
Now is there any function has been added into the appium flutter driver to get  the lists of all such ids/keys? And execute a common action on it , Or could please provide any other alternate ways to do so

**Comment by dbudim** [comment]
any solutions? 
Could it be implemented?
